hacep/src/main/java/org/kie/hacep/core/infra/consumer/DefaultKafkaConsumer.java

```diff
@@ -357,20 +360,23 @@ private void defaultProcessAsAReplica(int size) {
                 ConsumerRecord<String, T> first = records.iterator().next();
                 eventsBuffer = records.records(new TopicPartition(first.topic(), first.partition()));
                 consumeEventsFromBufferAsAReplica();
+            }else{
```

Comment:
Please add spaces to follow correct code style.

Reply:
Ok